tests/unit/test_configuration.py

```diff
@@ -43,19 +77,21 @@ def test_default_configuration_structure():
     for dataset in DATASETS:
         # Check row noise
         for row_noise in dataset.row_noise_types:
-            config_probability = config[dataset.name][Keys.ROW_NOISE][row_noise.name][
-                Keys.ROW_PROBABILITY
-            ]
-            default_probability = (
-                DEFAULT_NOISE_VALUES.get(dataset.name, {})
-                .get(Keys.ROW_NOISE, {})
-                .get(row_noise.name, {})
-                .get(Keys.ROW_PROBABILITY, "no default")
-            )
-            if default_probability == "no default":
-                assert config_probability == row_noise.row_probability
-            else:
-                assert config_probability == default_probability
+            config_probability = config[dataset.name][Keys.ROW_NOISE][row_noise.name]
```

Comment:
Row noise configuration is now structured identically to column noise configuration. I would expect the testing to look the same (and for it be possible to restructure this such that the code for column noise is reused for row noise. Is there a reason that this is not the case?